src/main/java/mil/dds/anet/threads/AnetEmailWorker.java

```diff
@@ -94,7 +97,11 @@ protected void runInternal(Instant now, JobHistory jobHistory, Map<String, Objec
         }
         processedEmails.add(email.getId());
       } catch (Throwable t) {
-        logger.error("Error sending email", t);
+        logger.atError().setCause(t).setMessage("Error sending email #{}")
+            .addArgument(email.getId()).log();
```

Comment:
Why not simply: `logger.error("Error sending email {}", email.getId(), t);` ?

Reply:
That wouldn't be the same thing. The only method with a particular `Throwable` argument is [this one](https://www.slf4j.org/apidocs-2.1.0/src-html/org/slf4j/Logger.html#line-814). Simply adding the `Throwable` as the final argument in a List treats it like any regular `Object`. https://www.slf4j.org/apidocs-2.1.0/src-html/org/slf4j/Logger.html#line-789

One thing I noticed though is that I'd really prefer for `Spotless` to align this call chain in a different way. Is there room for discussion on this?

Reply:
A Throwable as the last argument gets special treatment: https://www.slf4j.org/faq.html#paramException